injection/sharedmain/main.go

```diff
@@ -67,6 +68,21 @@ func GetConfig(masterURL, kubeconfig string) (*rest.Config, error) {
 	return nil, fmt.Errorf("could not create a valid kubeconfig")
 }
 
+// GetLoggingConfig gets the logging config from either the file system if present
+// or via reading a configMap from the API.
+// The context is expected to be initialized with injection.
+func GetLoggingConfig(ctx context.Context) (*logging.Config, error) {
+	loggingConfigData, err := configmap.Load("/etc/config-logging")
+	if err == nil {
+		return logging.NewConfigFromMap(loggingConfigData)
+	}
```

Comment:
@dprotaso  Why do we need both?  It should be safe to move from the volume to an API call, and then drop the volumes.  What am I missing? :)

Reply:
> Why do we need both?

We don't. I realize the other configs aren't mounted anymore